src/current/v23.1/goldengate.md

```diff
@@ -129,10 +129,10 @@ This section describes how to configure Oracle GoldenGate for PostgreSQL to work
 [Extract](https://docs.oracle.com/goldengate/c1230/gg-winux/GGCON/processes-and-terminology.html) is Oracle GoldenGate's data capture mechanism that is configured to run against the source database.
 
 {{site.data.alerts.callout_success}}
-The steps in this section should be run on a machine and in a directory where Oracle GoldenGate for Oracle is installed.
+Complete the steps in this section on a machine and in a directory where Oracle GoldenGate for Oracle is installed.
 {{site.data.alerts.end}}
 
-1. Inside the GGSCI terminal, create and open the `epos` parameter file for the Oracle source:
+1. In the GGSCI terminal, create and open the `epos` parameter file for the Oracle source:
```

Comment:
@nickvigilante oxford comma false positive.

Reply:
Vale has been disabled for all new PRs.